### ISSUE TITLE
Replace NWM High Magnitude color scheme

### ISSUE THREAD

**Issue by ChipGobs-NOAA** [NOAA-OWP/hydrovis#511]
Short description explaining the high-level reason for the new issue.

## Current behavior
NWM high magnitude stream lines uses AHPS-style colors

## Expected behavior

Use the following colors for both Analysis and MRF  - GFS

2% - #FF00E5
4% - #9E00FF
10% - #1400FF
20% - #2EC6FF
50% - #39EFFF
> 50% & > High Water Threshold - #B4FFF2
Insufficient Data - #BDC2BB

**Comment by shawncrawley** [comment]
ATTN @CoreyKrewson-NOAA:

@ChipGobs-NOAA has brought this up in the last couple of scrums. He said its coming down to crunch time pretty soon to get this moving. Ideally we could get in into Staging ASAP for it to be looked at and played with. Chip is working with @cfitzpat479 and WPOD to come up with an acceptable Prod release date.

I am fine working on this - it will be super quick, right? Just a mapx change? But I'm concerned about moving it into staging if we don't want it to actually be deployed with this upcoming push to Production. Is this something that we could even just modify in TI on the mapx, but then temporarily hard-code into UAT by making the same mapx tweak in-place and then recreating the SD to have the service recreated?

Let me know what you think.

P.S. Calling out @sudhirshrestha-noaa so he's in the loop (as requested by Chip)

**Comment by CoreyKrewson-NOAA** [comment]
yeah I share your same concerns about moving it into staging with the upcoming production push. I am fine with you making the change in UAT manually temporarily. The end of next week, after we do our prod push, we can push this to staging through TF

**Comment by shawncrawley** [comment]
@ChipGobs-NOAA I need some clarification on this. There are currently 9 high_flow_magnitude services (listed in the screnshot below). I know you documented above that this change should only apply to "Analysis and MRF - GFS" but even then, does this mean every analysis implementation, including Past 14-Day, Hawaii, and PRVI? It seems odd to me that we wouldn't change the symbology for all of them, since that would lead to mixed messaging and confusion between configurations and whatnot. Anyway, let me know and I will proceed to make the changes in UAT/Staging.

![image](https://github.com/NOAA-OWP/hydrovis/assets/12499203/efe04cb9-2189-42da-b78d-5a00621b7af7)

**Comment by CoreyKrewson-NOAA** [comment]
We should change all HFM services to have the same symbology

**Comment by shawncrawley** [comment]
@ChipGobs-NOAA All high_flow_magnitude services on UAT have been updated per your request. Please take a look at the services and let me know if you have any additional feedback. Here's a link to quickly get to them all in the UAT Portal content explorer:

https://maps-staging.water.noaa.gov/portal/home/content.html?view=grid&sortOrder=desc&sortField=modified&searchTerm=high_flow_magnitude#organization

**Comment by ChipGobs-NOAA** [comment]
Some of the legend colors look mostly the same (10,20,50 %).  They are red
and are supposed to be in the blue range.

[image: image.png]

**Comment by shawncrawley** [comment]
@ChipGobs-NOAA I finally found out what was wrong. The services should look good now. Let me know!
